### PR TITLE
Edit layout index course and header admin

### PR DESCRIPTION
**Pull request by lannp**

### CODE REVIEW

File: app/views/layouts/rails_admin/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <meta content="NONE,NOARCHIVE" name="robots"/>
+    <%= csrf_meta_tag %>
+    <%= stylesheet_link_tag "rails_admin/rails_admin.css", media: :all %>
+    <%= javascript_include_tag "rails_admin/rails_admin.js" %>
+    <% # Initialize JS simple i18n %>
+    <script>
+      
+      RailsAdmin.I18n.init('#{I18n.locale}', JSON.parse("#{j I18n.t("admin.js").to_json}"))
+
```
Comment (trantube94) [inline app/views/layouts/rails_admin/application.html.erb]: remove line

File: app/views/layouts/rails_admin/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    </script>
+  </head>
+  <body class="rails_admin">
+    <div class="label label-warning" id="loading" style="display:none; position:fixed; right:20px; bottom:20px; z-index:100000">
+      <%= t('admin.loading') %>
```
Comment (trantube94) [inline app/views/layouts/rails_admin/application.html.erb]: remove (), ' -> "

File: app/views/layouts/rails_admin/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <%= javascript_include_tag "rails_admin/rails_admin.js" %>
+    <% # Initialize JS simple i18n %>
+    <script>
+      
+      RailsAdmin.I18n.init('#{I18n.locale}', JSON.parse("#{j I18n.t("admin.js").to_json}"))
+
+    </script>
+  </head>
+  <body class="rails_admin">
+    <div class="label label-warning" id="loading" style="display:none; position:fixed; right:20px; bottom:20px; z-index:100000">
```
Comment (trantube94) [inline app/views/layouts/rails_admin/application.html.erb]: style nen cho vao file css

File: app/views/layouts/rails_admin/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      <%= t('admin.loading') %>
+    </div>
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <%= render "layouts/rails_admin/navigation" %>
+    </nav>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-1">
+          <div class="content" data-pjax-container="<%= true %>">
+            <%= render template: 'layouts/rails_admin/pjax' %>
```
Comment (trantube94) [inline app/views/layouts/rails_admin/application.html.erb]: ' -> "

**Comment by vietanhbkaaa** [comment]
OK

**Comment by hienbx94** [comment]
Ok

File: config/routes.rb [hunk 5]
```diff
@@ -14,3 +14,4 @@
   resources :subjects, only: [:show]
   resources :tasks, only: [:show]
 end
+
```
Comment (trantube94) [inline config/routes.rb]: remove line

File: app/assets/stylesheets/rails_admin/custom/theming.scss [hunk 6]
```diff
@@ -96,3 +96,10 @@
     }
   }
 }
+.body-rails-admin{
```
Comment (trantube94) [inline app/assets/stylesheets/rails_admin/custom/theming.scss]: add space before { }

**Comment by trantube94** [comment]
OK

File: app/models/concerns/rails_admin_course.rb [hunk 7]
```diff
@@ -13,8 +13,22 @@ module RailsAdminCourse
...
+          pretty_value do
+            status = bindings[:object]
+            class_label = if status.init?
+              "label label-success"
+            elsif status.progress?
+              "label label-info"
+            else status.finish?
+              "label label-default"
+            end
+            %{<div class="#{class_label}">#{status.status}</div >}.html_safe
```
Comment (hellovietnam93) [inline app/models/concerns/rails_admin_course.rb]: bỏ cái dấu cách đi e

**Pull request merged**